runners/direct-java/src/main/java/org/apache/beam/runners/direct/ForwardingPTransform.java

```diff
@@ -57,6 +57,6 @@ public String getName() {
 
   @Override
   public void populateDisplayData(DisplayData.Builder builder) {
-    delegate().populateDisplayData(builder);
+    builder.delegate(delegate());
```

Comment:
The important difference is to ensure the display data namespace is set correctly. In the previous syntax, the namespace would be `org.apache.beam.runners.direct.ForwardingPTransform`; with the new syntax, it is properly set to the delegate.

The only time a component should call `popluateDisplayData()` directly is on it's base class  via `super.populateDisplayData(builder)`. I believe this CL fixes up all existing instances.